exercises/roman-numerals/canonical-data.json

```diff
@@ -1,74 +1,92 @@
 {
    "cases": [
       {
+         "description": "1 is a single I",
          "number" : 1, 
          "expected": "I"
       },
       {
+         "description": "2 is two I's",
          "number" : 2, 
          "expected": "II"
       },
       {
+         "description": "3 is three I's",
          "number" : 3, 
          "expected": "III"
       },
       {
+         "description": "4, being 5 - 1, is IV",
          "number" : 4, 
          "expected": "IV"
       },
       {
+         "description": "5 is a single V",
          "number" : 5, 
          "expected": "V"
       },
       {
+         "description": "6, being 5 + 1, is VI",
          "number" : 6, 
          "expected": "VI"
       },
       {
+         "description": "9, being 10 - 1, is IX",
          "number" : 9, 
          "expected": "IX"
       },
       {
+         "description": "20 is two X's",
          "number" : 27, 
          "expected": "XXVII"
       },
       {
+         "description": "48 is not 50 - 2 but rather 40 + 8",
          "number" : 48, 
          "expected": "XLVIII"
       },
       {
+         "description": "50 is a single L",
          "number" : 59, 
          "expected": "LIX"
       },
       {
+         "description": "90, being 100 - 10, is XC",
          "number" : 93, 
          "expected": "XCIII"
       },
       {
+         "description": "100 is a single C",
          "number" : 141, 
          "expected": "CXLI"
       },
       {
+         "description": "60, being 50 + 10, is LX",
```

Comment:
I decided that the only new thing this case was testing was 60, but it's a bit of an illogical place to do that. It can be done earlier (near 50 and 90), without dealing with 100s. But I tried not to change tests in this PR.